default.py

```diff
@@ -97,8 +97,16 @@ def play(self, item):
                     stream['url'] += '|%s=%s' % (header, stream['headers'][header])
             print 'Sending %s to player' % stream['url']
             li = xbmcgui.ListItem(path=stream['url'], iconImage='DefaulVideo.png')
+            
+            sub = False
+            if xbmcaddon.Addon('xbmc.addon').getAddonInfo('version') > "16":
+                sub = True
+                if stream['subs'] != None and stream['subs'] != '':
+                    li.setSubtitles([stream['subs']])
+                
             xbmcplugin.setResolvedUrl(int(sys.argv[1]), True, li)
-            xbmcutil.load_subtitles(stream['subs'])
+            if sub == False:
+                xbmcutil.load_subtitles(stream['subs'])
```

Comment:
Tak to nepujde dat do load_subtitles, aby se zachovala zpetna kompatibilita, protoze po starem zpusobu se musi nacitat titulky az po volani setResolvedUrl, ale pri novem zpusobu pred volanim setResolvedUrl